beginners/1_blog.md

```diff
@@ -7,8 +7,9 @@ Why should you blog?
 - wordpress, medium or blogspot
 
 ### First post, what to write about?
+- Why Computer Science.
```

Comment:
This should be more detailed, with examples.

Reply:
Can you pull the recent changes and elaborate the same?